.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -230,7 +230,7 @@ steps:
     displayName: "Install azure-cli"
 
   - script: |
-      set -e
+      set -x
```

Comment:
Why make this change?

Reply:
I change this because I want to get more information in our roll out stage. We can clearly get the parameter passed into the test plan from console by changing this.